### ISSUE TITLE
Display information to the end user 

### ISSUE THREAD

**Issue by 12rambau** [12rambau/gwb#11]
map or table when relevant
I need to talk with Peter Vogt to have the complete list of the displayable elements :

- [ ] acc : 
- [ ] dist : 
- [ ] fad : 
- [ ] frag :
- [ ] lm :
- [ ] mspa :
- [ ] p222 : 
- [ ] rss : 
- [ ] spa :

**Comment by vogtpet** [comment]
update p222 to p223. _parc_ and _rec_ is missing in that list. You don't use GWB_REC in the UI but is is available via the cmd-line, if people want to use that. See output of the cmd: GWB

**Comment by 12rambau** [comment]
> You don't use GWB_REC in the UI but is is available via the cmd-line, if people want to use that. See output of the cmd: GWB

of course, that's why it is still referenced in the cli document I shared with you (https://github.com/openforis/sepal-doc/blob/master/docs/source/cli/gwb.rst)

**Comment by 12rambau** [comment]
@vogtpet for the element in the list that contains a tif I will display them on a map 

For the one with only a csv I will present the csv vertically as there is 1 sing le image to display
